source/manual/howto-change-a-government-following-an-election.html.md

```diff
@@ -25,19 +24,27 @@ Governments are listed at the [government path of Whitehall Admin][].
 2. Close the government.
 
 This will close the current government and remove all [ministerial
-appointments][]. Changes will instantly appear on the live site.
+appointments][]. The Content Support team will be able to re-assign ministers
+once the new government has been opened.
 
-The Content Support team will be able to re-assign ministers once the
-new government has been opened.
+This will cause a high number of documents to be represented by the Publishing
+API which might mean some delays on content being up to date on the website.
+Previous tests have shown that it takes around 3.5 hours for the queues to
+clear in integration.
+
+Grafana monitoring for:
+
+* [Publishing API](https://grafana.integration.publishing.service.gov.uk/dashboard/file/publishing-api.json?refresh=5s&orgId=1&from=now-6h&to=now)
```

Comment:
Should these link to Integration? Is this for testing? Obviously changing governments for real will be done in Production - I'd have thought those graphs would be more useful?

Reply:
Yeah, I was wondering that myself. I decided to leave it since the links were integration before.

Perhaps we could include two links? I imagine integration ones are still useful for testing.

Reply:
Two sets of links would work, yeah.

Reply:
I've added links for each environment now!